### PR TITLE
Add SetQueueAttributesV1 for JSON support

### PR DESCRIPTION
**Pull request by Admiral-Piett**

### CODE REVIEW

**Comment by Admiral-Piett** [comment]
@andrew-womeldorf I rebased and now I'm getting some race condition-y/index out of bound errors in my tests.  How did you fix your issues in particular?  I couldn't see much from the github history with all the squashing and things.

**Comment by andrew-womeldorf** [comment]
Interesting. The tests still pass fine on `feature-aws-json`, but I can reproduce the error locally when I checkout this branch. Looking...

**Comment by andrew-womeldorf** [comment]
Something is happening, though. If I comment out the failing test entirely, then the next test in that file fails with a similar cause.

**Comment by andrew-womeldorf** [comment]
If I reset the `utils.REQUEST_TRANSFORMER = utils.TransformRequest` at the top of the test, everything passes like normal. I'm assuming some test isn't resetting that properly. Looking...

File: app/gosqs/list_queues_test.go [hunk 1]
```diff
@@ -99,13 +99,7 @@ func TestListQueuesV1_success_with_queue_name_prefix_no_matching_queues(t *testi
...
 
 func TestListQueuesV1_request_transformer_error(t *testing.T) {
-	//conf.LoadYamlConfig("../conf/mock-data/mock-config.yaml", "BaseUnitTests")
-	defer func() {
-		utils.ResetApp()
-		utils.REQUEST_TRANSFORMER = utils.TransformRequest
-	}()
-
-	utils.REQUEST_TRANSFORMER = func(resultingStruct interfaces.AbstractRequestBody, req *http.Request) (success bool) {
+	utils.REQUEST_TRANSFORMER = func(resultingStruct interfaces.AbstractRequestBody, req *http.Request, emptyRequestValid bool) (success bool) {
```
[the remark this answers is not in the document]
Comment (Admiral-Piett) [reply app/gosqs/list_queues_test.go:102]: @andrew-womeldorf Ok, thanks!  That was my solution too.  I was just curious if there were any other nuances you came across during testing.  I'll sort this out shortly.  Thank you!

**Pull request merged**